apps/wallet/src/ui/app/components/navigation/index.tsx

```diff
@@ -6,21 +6,25 @@ import cl from 'classnames';
 import { memo } from 'react';
 import { NavLink } from 'react-router-dom';
 
+import { useActiveAccount } from '../../hooks/useActiveAccount';
+import { useIsAccountReadLocked } from '../../hooks/useIsAccountReadLocked';
 import { useAppSelector } from '_hooks';
 import { getNavIsVisible } from '_redux/slices/app';
 
 import st from './Navigation.module.scss';
 
-function makeLinkCls({ isActive }: { isActive: boolean }) {
-	return cl(st.link, { [st.active]: isActive });
-}
-
 export type NavigationProps = {
 	className?: string;
 };
 
 function Navigation({ className }: NavigationProps) {
 	const isVisible = useAppSelector(getNavIsVisible);
+	const activeAccount = useActiveAccount();
+	const isActiveAccountReadLocked = useIsAccountReadLocked(activeAccount);
+	const makeLinkCls = ({ isActive }: { isActive: boolean }) =>
+		cl(st.link, { [st.active]: isActive, [st.disabled]: isActiveAccountReadLocked });
+	const makeLinkClsNoDisabled = ({ isActive }: { isActive: boolean }) =>
```

Comment:
name here is a little confusing 😅 but we can eventually fix / remove the CSS module file

Reply:
yeah we would update this component at some point